surveys/2024-04/questions.md

```diff
@@ -0,0 +1,160 @@
+
+- Which of the current [Technical Priorities](https://github.com/nodejs/node/blob/main/doc/contributing/technical-priorities.md) are important to you ? (multiple choice)
+  - Modern HTTP
+  - Suitable types for end-users
+  - Documentation
+  - WebAssembly
+  - ESM
+  - Support for features from the latest ECMAScript spec
+  - Observability
+  - Permissions/policies/security model
+  - Better multithreaded support
+  - Single Executable Applications
+
+- Are there technical priorities that you believe are missing (open question)
+
+- What is important to you ? (multiple choice)
+  - Good understanding of the direction of the project
+  - Ability to affect the direction of the project
+  - Consumable APIs and docs
+  - Predictable and stable releases
+  - Innovation at a consumable pace
+  - Easy Installation
+  - Easy issue reporting, resolution and collaboration
+  - Broad deployment platform support
+  - Broad desktop platform support
+  - Consistent and intuitive error handling
+  - Runtime diagnostic tooling
+  - Development time diagnostic tooling
+  - Relevant APIs in core
+  - Module/dependency info and management
+  - Ways to fund their work
+  - Ability to assess impact of changes they make
+  - Reasonable resource usage/performance
+  - Good security and CVE practices in the project
+  - Better CVE management in the ecosystem
+  - Good CI infrastructure and experience in the project
+  - Supportive Collaborators and Environment in the project
+  - Better ways to build consensus in the project
+  - Easy contribution workflow
+  - Ability to embed and bundle the Node.js runtime
+  - A well maintained and secure standard library
+  - Assets that show Node.js is a good choice
+
+- What is important to you that is not in this list? (open question)
+
+## Technical Questions
+
+- Regardless of how your code is written in its original form, when it runs in production, does it contain ES module syntax (`import`/`export`)?
+  - Yes
+  - No
+
+- For those of you wishing to use ESM in an existing application, what have been the pain points or blockers preventing you from doing so (if any)? (open question)
+
+- Are you using the following experimental features of Node.js (multiple choice)?
+  - Corepack
+  - Async Hooks
+  - Permission model / Policies
+  - Single Executable Application
+  - Startup Snapshot
+  - Loader hooks (`--loader` or `module.register()`)
+  - Network import (`import 'http://...'`)
+  - WASI
+  - VM Modules (`--experimental-vm-modules`)
+  - Watch mode (`--watch`)
+  - WebSocket client
+  - Trace events (`--trace-event-categories` or `node:trace_events`)
+  - Dotenv (`--env-file`)
+  - Import attributes
+
+- Are you using the following new stable features (multiple choice):
+- Test runner (`node:test`)
+- Web Crypto (`globalThis.crypto` or `crypto.webcrypto`)
+- Web Streams (`node:stream/web`)
+- Fetch
+
+- Do you encounter any recurring issues when using Node.js that you would like to share with us ? (open question)
```

Comment:
Do we want to add a question about the governance of the project? I think we discussed that

Reply:
``` 
Are you aware of the Node.js goverance model? 
```

Reply:
I thought we were trying to cut questions, not add them. I don’t see the need for this one. We have a free response section if anyone wants to complain about our governance.